panda/src/putil/bamReader.cxx

```diff
@@ -133,6 +133,18 @@ init() {
     _file_stdfloat_double = scan.get_bool();
   }
 
+  if(_file_minor >= 46) {
+    _tex_compression_format = (BamTexCompressionFormat)scan.get_uint8();
+  }
+
+  if(_tex_compression_format == BTC_zlib) {
+#ifndef HAVE_ZLIB
```

Comment:
Maybe the guard should be moved to encompass the entire `if(_tex_compression_format == BTC_zlib)` block?